test-utils/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
     api(project(":model"))
 
     api("io.kotest:kotest-core-jvm:$kotestVersion")
+    api("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
```

Comment:
Does it make sense to also file this as a feature request for upstream kotest?

Reply:
Yes, I'm already looking into it.